docs/configuration/ssl.md

```diff
@@ -136,44 +136,9 @@ Certain versions of nginx have bugs that prevent [HTTP/2](https://nginx.org/en/d
 
 Your application has access to the HTTP headers `X-Forwarded-Proto`, `X-Forwarded-Port` and `X-Forwarded-For`. These headers indicate the protocol of the original request (HTTP or HTTPS), the port number, and the IP address of the client making the request, respectively. The default configuration is for Nginx to set these headers.
 
-If your server runs behind an HTTP(S) load balancer, then Nginx will see all requests as coming from the load balancer. If your load balancer sets the `X-Forwarded-` headers, you can tell Nginx to pass these headers from load balancer to your application by using a [custom nginx template](/docs/configuration/nginx.md#customizing-the-nginx-configuration). The following is a simple example of how to do so.
-
-```go
-server {
-  listen      [::]:{{ .PROXY_PORT }};
-  listen      {{ .PROXY_PORT }};
-  server_name {{ .NOSSL_SERVER_NAME }};
-  access_log  /var/log/nginx/{{ .APP }}-access.log;
-  error_log   /var/log/nginx/{{ .APP }}-error.log;
-
-  location    / {
-    proxy_pass  http://{{ .APP }};
-    proxy_http_version 1.1;
-    proxy_set_header Upgrade $http_upgrade;
-    proxy_set_header Connection $http_connection;
-    proxy_set_header Host $http_host;
-    proxy_set_header X-Forwarded-Proto $http_x_forwarded_proto;
-    proxy_set_header X-Forwarded-For $http_x_forwarded_for;
-    proxy_set_header X-Forwarded-Port $http_x_forwarded_port;
```

Comment:
Should we do anything about all the other properties here? Would it make sense to instead make these values toggleable via an `nginx:set` call?

Reply:
I don't think so. Only the value in `-For` is appended. `-Proto` and `-Port` are being replaced:

```bash
curl -H "X-Forwarded-Port: 1234" https://...

  ["HTTP_X_FORWARDED_PORT"]=>
  string(3) "443"
```

```bash
curl -H "X-Forwarded-Proto: Banana" https://...

  ["HTTP_X_FORWARDED_PROTO"]=>
  string(3) "https"
```

Reply:
Yeah just thinking that removing this doc maybe isn't the best. Maybe we can add properties for each of these headers as well?

Reply:
I'm not sure if I understand you fully. I will undo the removal of the paragraph "Only use this option if ..." and the paragraph after. Do you also want to undo the deletion of the custom template example? I did add an example of how to configure via `nginx:set`, which makes more sense I think.

Regarding making `-Proto` and `-Port` toggleable: I can do it, but I'm not sure if its a common use case. Also not sure between which values you should be able to toggle.

Reply:
If you're around on the gliderlabs slack, I'd be happy to jump on a quick call to explain what I mean. I'll be around for the next ~30 minutes.